src/commons/buttons/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+const Button = ({ text, onClick, type, disabled, className, id }) => (
+  <button
+    type={type}
+    disabled={disabled}
+    id={id}
+    onClick={onClick}
+    className={className}
+  >
+    {text}
+  </button>
+);
+export default Button;
```

Comment:
Great work @wasibani-roy, however, add a line at the end of this file

Reply:
Thank you added the line

Reply:
Great